content/master/software/install.md

```diff
@@ -11,9 +11,15 @@ If you don't have a Kubernetes cluster create one locally with [Kind](https://ki
 {{< /hint >}}
 
 ## Prerequisites
-* [Kubernetes](https://kubernetes.io/releases/) version `v1.16.0` or later
+* [Kubernetes](https://kubernetes.io/releases/) version `v1.23.0` or later
```

Comment:
How good do you feel about keeping up with Kubernetes releases and bumping this version? Would the statement that we support what Kubernetes supports (with a link to their supported versions) be enough?

Reply:
Great point. Opened a new PR for that. https://github.com/crossplane/docs/pull/438